WordPress/src/main/java/org/wordpress/android/ui/posts/PrepublishingBottomSheetFragment.kt

```diff
@@ -37,6 +36,7 @@ class PrepublishingBottomSheetFragment : WPBottomSheetDialogFragment(),
         PrepublishingScreenClosedListener, PrepublishingActionClickedListener {
     @Inject internal lateinit var viewModelFactory: ViewModelProvider.Factory
     @Inject internal lateinit var analyticsTrackerWrapper: AnalyticsTrackerWrapper
+    @Inject internal lateinit var navBarModeHelper: NavigationBarInteractionModeHelper
```

Comment:
It looks like `navBarModeHelper` is now unused. In fact, it looks like we can drop the `NavigationBarInteractionModeHelper` class entirely now.

Reply:
This is now removed in 5fef752